Articles of Association.md

```diff
@@ -0,0 +1,57 @@
+        10. Election of 3-7 board members and 2 substitutes
+        11. Election of auditor (s) and deputy auditor 
+        12. Any other business
+    6. Proposals that are to be considered at the General Assembly must be received by the Board of Directors no later than 8 days before the General Assembly. Proposals for amendments to the articles of association that are to be considered must be received by the Board of Directors no later than three weeks before the General Assembly.
+    7. The General Assembly is presided by the Speaker who cannot be a member of the Board of Directors.
+    8. The General Assembly makes its resolutions by 1/2 of votes and by electronic voting.
+    9. Voting can be done anonymously if 1/3 of the Board of Directors is in favour.
+5. Extraordinary General Assembly
+    1. Extraordinary General Assembly may be held when the Board of Directors deems it necessary or when at least 1/3 of the members submit a written reasoned request to the chairman and deputy chairman. In such cases, the General Assembly must be held no later than four weeks after the request has come to the knowledge of the chairman or deputy chairman.
+6. The daily management of the association
+    1. The daily management of the association consists of the board, which in addition to the chairman consists of the deputy chairman, the treasurer and two to six additional members. The board of directors is elected by the General Assembly.
+    2. The board of directors leads the association in accordance with these articles of association and the resolutions of the General Assembly.
+    3. No later than 14 days after re-electing the Board of Directors, a constituent meeting is held, where the Board of Directors elects the treasurer from among its members. All positions of trust are valid until the next General Assembly.
+    4. The Board of Directors also determines its own rules of procedure. It may set up subcommittees and working groups to carry out specific tasks.
+    5. The chairman - and in their absence the deputy chairman - convenes and chairs the meetings of the board. Convening is done in writing stating the agenda when the chairman deems it necessary, or at least three members of the Board of Directors make a request to the chairman. In such cases, the meeting shall be held no later than 2 weeks after the request has come to the knowledge of the chairman.
+7. Finance, accounting and auditing
+    1. The association's financial year follows the calendar year.
+    2. The Board of Directors is responsible to the General Assembly for the budget and accounts.
+    3. The association's accounts are kept by the treasurer, who also keeps the association's membership register.
+    4. The accounts are audited by the auditor (s) and deputy auditor elected at the General Assembly.
+8. Signing authority
+    1. The association is signed externally by the signature of the chairman and at least one additional board member in the association.
```

Comment:
"signed externally" is still very weird wording but I am not quite sure how to improve that;

also the deputy chairman should probably be allowed in place of the chairman in case of absence right?

Reply:
That the deputy chairman can stand in for the chairman is a given.